// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/declarationEmitUsingAlternativeContainingModules1.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,6 +1,6 @@
 
 //// [src/index.d.mts.map]
-{"version":3,"file":"index.d.mts","sourceRoot":"","sources":["index.mts"],"names":[],"mappings":"AAIA,UAAU,MAAM;IACd,GAAG,EAAE,MAAM,CAAA;IACX,WAAW,EAAE,MAAM,CAAA;IACnB,IAAI,EAAE,MAAM,CAAA;IACZ,KAAK,EAAE,OAAO,CAAA;IACd,IAAI,EAAE,MAAM,CAAA;IACZ,IAAI,EAAE,MAAM,CAAA;IACZ,QAAQ,EAAE,MAAM,CAAA;CACjB;AAgBD,eAAO,MAAM,UAAU,2CAMtB,CAAA"}
+{"version":3,"file":"index.d.mts","sourceRoot":"","sources":["index.mts"],"names":[],"mappings":"AAIA,UAAU,MAAM;IACd,GAAG,EAAE,MAAM,CAAA;IACX,WAAW,EAAE,MAAM,CAAA;IACnB,IAAI,EAAE,MAAM,CAAA;IACZ,KAAK,EAAE,OAAO,CAAA;IACd,IAAI,EAAE,MAAM,CAAA;IACZ,IAAI,EAAE,MAAM,CAAA;IACZ,QAAQ,EAAE,MAAM,CAAA;CACjB;AAgBD,eAAO,MAAM,UAAU,QAAO,kBAAkB,CAAC,MAAM,EAAE,EAAE,KAAK,CAM/D,CAAA"}
 
-//// https://sokra.github.io/source-map-visualization#base64,aW50ZXJmYWNlIElFbnRyeSB7DQogICAgQVBJOiBzdHJpbmc7DQogICAgRGVzY3JpcHRpb246IHN0cmluZzsNCiAgICBBdXRoOiBzdHJpbmc7DQogICAgSFRUUFM6IGJvb2xlYW47DQogICAgQ29yczogc3RyaW5nOw0KICAgIExpbms6IHN0cmluZzsNCiAgICBDYXRlZ29yeTogc3RyaW5nOw0KfQ0KZXhwb3J0IGRlY2xhcmUgY29uc3QgdXNlRW50cmllczogKCkgPT4gVXNlUXVlcnlSZXR1cm5UeXBlPElFbnRyeVtdLCBFcnJvcj47DQpleHBvcnQge307DQovLyMgc291cmNlTWFwcGluZ1VSTD1pbmRleC5kLm10cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC5tdHMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJpbmRleC5tdHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBSUEsVUFBVSxNQUFNO0lBQ2QsR0FBRyxFQUFFLE1BQU0sQ0FBQTtJQUNYLFdBQVcsRUFBRSxNQUFNLENBQUE7SUFDbkIsSUFBSSxFQUFFLE1BQU0sQ0FBQTtJQUNaLEtBQUssRUFBRSxPQUFPLENBQUE7SUFDZCxJQUFJLEVBQUUsTUFBTSxDQUFBO0lBQ1osSUFBSSxFQUFFLE1BQU0sQ0FBQTtJQUNaLFFBQVEsRUFBRSxNQUFNLENBQUE7Q0FDakI7QUFnQkQsZUFBTyxNQUFNLFVBQVUsMkNBTXRCLENBQUEifQ==,aW1wb3J0IHsgdXNlUXVlcnkgfSBmcm9tICdAdGFuc3RhY2svdnVlLXF1ZXJ5JwoKY29uc3QgYmFzZVVybCA9ICdodHRwczovL2FwaS5wdWJsaWNhcGlzLm9yZy8nCgppbnRlcmZhY2UgSUVudHJ5IHsKICBBUEk6IHN0cmluZwogIERlc2NyaXB0aW9uOiBzdHJpbmcKICBBdXRoOiBzdHJpbmcKICBIVFRQUzogYm9vbGVhbgogIENvcnM6IHN0cmluZwogIExpbms6IHN0cmluZwogIENhdGVnb3J5OiBzdHJpbmcKfQoKY29uc3QgdGVzdEFwaSA9IHsKICBnZXRFbnRyaWVzOiAoKTogUHJvbWlzZTxJRW50cnlbXT4gPT4gewogICAgcmV0dXJuIGZldGNoKGJhc2VVcmwgKyAnZW50cmllcycpCiAgICAgIC50aGVuKChyZXMpID0+IHJlcy5qc29uKCkpCiAgICAgIC50aGVuKChkYXRhKSA9PiBkYXRhLmVudHJpZXMpCiAgICAgIC5jYXRjaCgoZXJyKSA9PiBjb25zb2xlLmxvZyhlcnIpKQogIH0KfQoKY29uc3QgZW50cnlLZXlzID0gewogIGFsbDogWydlbnRyaWVzJ10gYXMgY29uc3QsCiAgbGlzdDogKCkgPT4gWy4uLmVudHJ5S2V5cy5hbGwsICdsaXN0J10gYXMgY29uc3QKfQoKZXhwb3J0IGNvbnN0IHVzZUVudHJpZXMgPSAoKTogVXNlUXVlcnlSZXR1cm5UeXBlPElFbnRyeVtdLCBFcnJvcj4gPT4gewogIHJldHVybiB1c2VRdWVyeSh7CiAgICBxdWVyeUtleTogZW50cnlLZXlzLmxpc3QoKSwKICAgIHF1ZXJ5Rm46IHRlc3RBcGkuZ2V0RW50cmllcywKICAgIHNlbGVjdDogKGRhdGEpID0+IGRhdGEuc2xpY2UoMCwgMTApCiAgfSkKfQo=
+//// https://sokra.github.io/source-map-visualization#base64,aW50ZXJmYWNlIElFbnRyeSB7DQogICAgQVBJOiBzdHJpbmc7DQogICAgRGVzY3JpcHRpb246IHN0cmluZzsNCiAgICBBdXRoOiBzdHJpbmc7DQogICAgSFRUUFM6IGJvb2xlYW47DQogICAgQ29yczogc3RyaW5nOw0KICAgIExpbms6IHN0cmluZzsNCiAgICBDYXRlZ29yeTogc3RyaW5nOw0KfQ0KZXhwb3J0IGRlY2xhcmUgY29uc3QgdXNlRW50cmllczogKCkgPT4gVXNlUXVlcnlSZXR1cm5UeXBlPElFbnRyeVtdLCBFcnJvcj47DQpleHBvcnQge307DQovLyMgc291cmNlTWFwcGluZ1VSTD1pbmRleC5kLm10cy5tYXA=,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguZC5tdHMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJpbmRleC5tdHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBSUEsVUFBVSxNQUFNO0lBQ2QsR0FBRyxFQUFFLE1BQU0sQ0FBQTtJQUNYLFdBQVcsRUFBRSxNQUFNLENBQUE7SUFDbkIsSUFBSSxFQUFFLE1BQU0sQ0FBQTtJQUNaLEtBQUssRUFBRSxPQUFPLENBQUE7SUFDZCxJQUFJLEVBQUUsTUFBTSxDQUFBO0lBQ1osSUFBSSxFQUFFLE1BQU0sQ0FBQTtJQUNaLFFBQVEsRUFBRSxNQUFNLENBQUE7Q0FDakI7QUFnQkQsZUFBTyxNQUFNLFVBQVUsUUFBTyxrQkFBa0IsQ0FBQyxNQUFNLEVBQUUsRUFBRSxLQUFLLENBTS9ELENBQUEifQ==,aW1wb3J0IHsgdXNlUXVlcnkgfSBmcm9tICdAdGFuc3RhY2svdnVlLXF1ZXJ5JwoKY29uc3QgYmFzZVVybCA9ICdodHRwczovL2FwaS5wdWJsaWNhcGlzLm9yZy8nCgppbnRlcmZhY2UgSUVudHJ5IHsKICBBUEk6IHN0cmluZwogIERlc2NyaXB0aW9uOiBzdHJpbmcKICBBdXRoOiBzdHJpbmcKICBIVFRQUzogYm9vbGVhbgogIENvcnM6IHN0cmluZwogIExpbms6IHN0cmluZwogIENhdGVnb3J5OiBzdHJpbmcKfQoKY29uc3QgdGVzdEFwaSA9IHsKICBnZXRFbnRyaWVzOiAoKTogUHJvbWlzZTxJRW50cnlbXT4gPT4gewogICAgcmV0dXJuIGZldGNoKGJhc2VVcmwgKyAnZW50cmllcycpCiAgICAgIC50aGVuKChyZXMpID0+IHJlcy5qc29uKCkpCiAgICAgIC50aGVuKChkYXRhKSA9PiBkYXRhLmVudHJpZXMpCiAgICAgIC5jYXRjaCgoZXJyKSA9PiBjb25zb2xlLmxvZyhlcnIpKQogIH0KfQoKY29uc3QgZW50cnlLZXlzID0gewogIGFsbDogWydlbnRyaWVzJ10gYXMgY29uc3QsCiAgbGlzdDogKCkgPT4gWy4uLmVudHJ5S2V5cy5hbGwsICdsaXN0J10gYXMgY29uc3QKfQoKZXhwb3J0IGNvbnN0IHVzZUVudHJpZXMgPSAoKTogVXNlUXVlcnlSZXR1cm5UeXBlPElFbnRyeVtdLCBFcnJvcj4gPT4gewogIHJldHVybiB1c2VRdWVyeSh7CiAgICBxdWVyeUtleTogZW50cnlLZXlzLmxpc3QoKSwKICAgIHF1ZXJ5Rm46IHRlc3RBcGkuZ2V0RW50cmllcywKICAgIHNlbGVjdDogKGRhdGEpID0+IGRhdGEuc2xpY2UoMCwgMTApCiAgfSkKfQo=
 
